lab2.js

```diff
@@ -77,8 +97,8 @@ assert(blob.hoursToOoze(1000, 1) === hoursSpentInDowington,
 
 var hello = {
   klingon: 'nuqneH',  // home planet is Qo'noS
-  romulan: 'Jolan\'tru', // home planet is Romulus
-  'federation standard': 'hello' // home planet is Earth
+  romulan: 'Jolan\"tru', // home planet is Romulus
+  federationStandard: 'hello' // home planet is Earth
```

Comment:
shouldn't need to change this - that's what bracket notation (for objects) is useful for